ci/edmtool.py

```diff
@@ -106,7 +106,6 @@
     "traitsui",
     "pyface",
     "pypdf2",
-    "swig",
     "unittest2",
```

Comment:
we don't depend on `unittest2` anymore. we should be able to remove this - maybe in a later PR